clang/test/CAS/debug-line-split.c

```diff
@@ -0,0 +1,12 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %clang -g -c -fcas-friendly-debug-info=debug-line-only -Xclang -fcas-backend -Xclang -fcas-backend-mode=native -Xclang -fcas-path -Xclang %t/cas %s -o %t/mccas_output.o
+// RUN: %clang -g -c -fcas-friendly-debug-info=debug-line-only %s -o %t/output.o
+// RUN: diff %t/output.o %t/mccas_output.o
```

Comment:
This is test is basically the same as run the clang once with `-Xclang -fcas-backend-mode=verify`.

I am hoping for a better test, e.g., run cas-backend, then dwarfdump the object file to make sure the debug line is correct.

Reply:
Yep you are completely right!